### PR TITLE
Removing SDK from LUIS endpoint calls

### PR DESCRIPTION
**Pull request by munozemilio**
Fixes #829, #834, #838

### CODE REVIEW

File: packages/luis/src/commands/luis/application/import.ts [hunk 1]
```diff
@@ -96,4 +87,18 @@ export default class LuisApplicationImport extends Command {
     await utils.writeUserConfig(userConfig, configDir)
   }
 
+  async formatInput(inputContent: string, name: string) {
+    let result = inputContent
+    try {
+      JSON.parse(inputContent)
+      /*tslint:disable: no-unused*/
+    } catch (error) {
+      let LuisObject = await Luis.fromContentAsync(inputContent)
```
Comment (vishwacsena) [inline packages/luis/src/commands/luis/application/import.ts:96]: Yay! :)

File: packages/luis/src/commands/luis/version/import.ts [hunk 2]
```diff
@@ -69,4 +58,19 @@ export default class LuisVersionImport extends Command {
     }
   }
 
+  async formatInput(inputContent: string, versionId: string) {
+    let result = inputContent
+    try {
+      JSON.parse(inputContent)
+      /*tslint:disable: no-unused*/
+    } catch (error) {
+      let LuisObject = await Luis.fromContentAsync(inputContent)
```
Comment (vishwacsena) [inline packages/luis/src/commands/luis/version/import.ts:67]: We should update doc to call out we support both JSON and LU

File: packages/dialog/test/commands/dialog/merge.test.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ describe('dialog:merge', async () => {
         process.chdir(srcDir)
     })
 
-    it('app.schema', async () => {
+    xit('app.schema', async () => {
```
[the remark this answers is not in the document]
Comment (munozemilio) [reply packages/dialog/test/commands/dialog/merge.test.ts:46]: Yes...Its failing on master

File: packages/luis/src/commands/luis/version/import.ts (hunk 2 above)
Comment (munozemilio) [reply packages/luis/src/commands/luis/version/import.ts:67]: It is already in the command description and it will be taken care once this is released

**Pull request merged**